.nvmrc

```diff
@@ -1 +1 @@
-4.8.4
\ No newline at end of file
+6.0.0
```

Comment:
It's not possible to use a recent version due to projects using an old one right?

Reply:
Yes, we should target the lowest supported version